dev/org.eclipse.codewind.core/src/org/eclipse/codewind/core/internal/InstallUtil.java

```diff
@@ -309,6 +328,55 @@ public static String getInstallerExecutable() throws IOException {
 		}
 	}
 	
+	protected static void copyExecutable(Map<OperatingSystem, String> fileMap) throws IOException {
+		// Get the current platform and choose the correct executable path
+		OperatingSystem os = PlatformUtil.getOS(System.getProperty("os.name"));
+		String relPath = fileMap.get(os);
+		
+		if (relPath == null) {
+			String msg = "Failed to get the relative path for the file executable";
+			Logger.logError(msg);
+			throw new IOException(msg);
+		}
+		
+		// Get the file path
+		String installerDir = getInstallerDir();
+		String execName = relPath.substring(relPath.lastIndexOf('/') + 1);
+		String execPath = installerDir + File.separator + execName;
+		
+		// Check if file exists, if it does return the path and skip the copying
+		if ((new File(execPath)).exists()) {
```

Comment:
You might also want to have a variable that indicates the file has been copied over.  If the user is using an old Eclipse workspace with a newer version of Codewind the file might exist in their workspace at an older level.